its/ruling/src/test/java/org/sonar/python/it/RulingHelper.java

```diff
@@ -143,6 +142,7 @@ static List<String> bugRuleKeys() {
       "S6323",
       "S6328",
       "S6468",
+      "S6554",
```

Comment:
We probably don't want to enable this rule on extended ruling since it's not a bug rule.
I'm torn on this one though, as it's a rule that is quite reliant on our semantics, and the extended ruling is about good semantic resolutions.

Reply:
done